google_chat_ros/README.md

```diff
@@ -0,0 +1,66 @@
+# The package for using Google chat services with ROS
+
+## What is this?
+Use Google Chat API with ROS.  
+![Screenshot from 2021-11-01 15-53-27](https://user-images.githubusercontent.com/27789460/139635911-66232c88-d3b9-4d7d-940e-966fbac9d800.png)  
+System components  
+![GoogleChatROS_system](https://user-images.githubusercontent.com/27789460/139635648-4ddbf9da-90e9-4b87-b958-ca996a8ffc4f.png)
```

Comment:
- please add `space`? `ID`? concept
- please add information of which port number should be open to internet.
- ->arrow include is strange, you can write big rectangle and put the API client code and ROS action server within that.
- how can we get message from other users in the chat room?

Reply:
> please add information of which port number should be open to internet

We don't have to open any ports.

Reply:
> how can we get message from other users in the chat room?

it is solved with https://github.com/jsk-ros-pkg/jsk_3rdparty/pull/242

Reply:
> it is solved with #242 .

This PR is ROS interface for GoogleChat API, so please implement all (or almost all) methods described in https://developers.google.com/chat/api/reference/rest,
Using dialogflow is JSK's application, but jsk_3rdparty is designed for general interface for existing libraries.

Reply:
sure, I'll cover them.